src/concepts/components/properties.md

```diff
@@ -1,26 +1,551 @@
 ---
-description: Parent to child communication
+description: The component could expose property attributes to receive data from the parent html component
 ---
 
 # Properties
 
-Properties enable child and parent components to communicate with each other.
+It is a common need to receive data from the parent component using properties.
 
-## Derive macro
+The properties are defined in a dedicated Rust struct derivating `Properties` and `Clone`traits.
 
-Don't try to implement `Properties` yourself, derive it by using `#[derive(Properties)]` instead.
+> This struct is usually named `Props`
 
-{% hint style="info" %}
-Types for which you derive `Properties` must also implement `Clone`. This can be done by either using `#[derive(Properties, Clone)` or manually implementing `Clone` for your type.
-{% endhint %}
+The properties bag is attached to the state adding the field `props` in the component struct and defining the `Properties` `Type` in the component trait implementation.
```

Comment:
You don't need to add a field `props` in the struct implementing `Component`.

Reply:
Mistake from my learning. I was following the # Optimizations & Best Practices. Where it is recommend to attach.
After few days ... I assumed it was necessary :-(

I will remove the reference to the concept of attaching. 

I will do it after lunch ... ;-)